openid-4-verifiable-presentations-1_0.md

```diff
@@ -811,14 +811,20 @@ Note: Some of the processing rules of the Presentation Definition and the Presen
 
 The Verifier has the choice of the following mechanisms to invoke a Wallet:
 
-- Custom URL scheme as an `authorization_endpoint` (for example, `openid4vp://` as defined in (#openid4vp-profile))
+- Custom URL scheme as an `authorization_endpoint` using `openid4vp://` as defined in (#openid4vp-custom-url-scheme)
```

Comment:
I'm worried this might make it look like only `openid4vp://` can be used, whereas, for example, 18013-7 iso mdl defines it's own custom scheme, `mdoc-openid4vp://`

Reply:
since there is space for more than a single custom url scheme using openid4vp, I am wondering if it would be an ARRAY instead of a string

while about the normative MUST, we might say that at least openid4vp:// must be included

Reply:
```suggestion
- Custom URL scheme as an `authorization_endpoint` (for example, `openid4vp://` as defined in (#openid4vp-profile))
```
+1 to Joseph's concern. please keep it as-is, or suggest a text that clarifies that openid4vp:// is not the only scheme that can be used.